go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21.12
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.12.1
-	github.com/ava-labs/avalanchego v1.11.11
+	github.com/ava-labs/avalanchego v1.11.11-rc.2
```

Comment:
Should this be `v1.11.12-rc.2` everywhere? i.e bump the patch from 11 to 12 and add the release candidate

Reply:
https://semvercompare.azurewebsites.net/?version=1.11.11&version=1.11.11-rc.2&version=1.11.12-rc.2